pvlib/temperature.py

```diff
@@ -978,3 +978,397 @@ def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
+    one of the use_* methods must be called to provide thermal model
+    parameters.
+
+    References
+    ----------
+    .. [1] A. Driesse et al, "PV Module Operating Temperature
+       Model Equivalence and Parameter Translation". 2022 IEEE
+       Photovoltaic Specialists Conference (PVSC), 2022.
+
+    Examples
+    --------
+    glm = GenericLinearModel(module_efficiency=0.19, absorptance=0.88)
+
+    glm.use_faiman(16, 8)
+
+    glm.to_pvsyst()
+
+    parmdict = glm.to_pvsyst()
+
+    pvsyst_cell(800, 20, 1, **parmdict)
+
+    See also
+    --------
+    pvlib.temperature.generic_linear
+    '''
+    # Contributed by Anton Driesse (@adriesse), PV Performance Labs, Sept. 2022
+
+    def __init__(self, module_efficiency, absorptance):
+
+        self.u_const = np.nan
+        self.du_wind = np.nan
+        self.eta = module_efficiency
+        self.alpha = absorptance
+        return None
+
+    def __repr__(self):
+
+        return self.__class__.__name__ + ': ' + vars(self).__repr__()
+
+    def __call__(self, poa_global, temp_air, wind_speed,
+                 module_efficiency=None):
+        '''
+        Calculate module temperature using the generic linear model and
+        previously initialized parameters.
+
+        Parameters
+        ----------
+        poa_global : numeric
+            Total incident irradiance [W/m^2].
+
+        temp_air : numeric
+            Ambient dry bulb temperature [C].
+
+        wind_speed : numeric
+            Wind speed in m/s measured at the same height for which the wind
+            loss factor was determined.  [m/s]
+
+        module_efficiency : numeric, optional
+            Module electrical efficiency.  The default value is the one
+            that was specified initially. [pu]
+
+        Returns
+        -------
+        numeric, values in degrees Celsius
+
+        See also
+        --------
+        get_generic
+        pvlib.temperature.generic_linear
+        '''
+        if module_efficiency is None:
+            module_efficiency = self.eta
+
+        heat_input = poa_global * (self.alpha - module_efficiency)
+        total_loss_factor = self.u_const + self.du_wind * wind_speed
+        temp_difference = heat_input / total_loss_factor
+
+        return temp_air + temp_difference
+
+    def get_generic_linear(self):
+        '''
+        Get the generic linear model parameters to use with the separate
+        generic linear module temperature calculation function.
+
+        Returns:
+        --------
+        model_parameters : dict
+
+        See also
+        --------
+        pvlib.temperature.generic_linear
+        '''
+        return dict(u_const=self.u_const,
+                    du_wind=self.du_wind,
+                    module_efficiency=self.eta,
+                    absorptance=self.alpha)
+
+    def use_faiman(self, u0, u1, **kwargs):
+        '''
+        Use the Faiman model parameters to set the generic equivalents.
```

Comment:
```suggestion
        Use the Faiman model parameters to set the generic_linear model equivalents.
```

Similar adjustment to other "use_" docstrings.